tests/test_fmi_master.py

```diff
@@ -206,6 +207,59 @@ def test_basic_simulation_txt_file_naming_exists(self):
         
         assert os.path.isfile("Test1.txt"), "Test1.txt does not exists"
         assert os.path.isfile("Test2.txt"), "Test2.txt does not exists"
+    
+    @testattr(stddist = True)
+    def test_basic_simulation_csv_file_naming_exists(self):
+        models, connections = self._load_basic_simulation()
+        
+        opts = {"result_handling":"csv", "result_file_name": {models[0]: "Test1.csv", models[1]: "Test2.csv"}}
+        
+        res = self._sim_basic_simulation(models, connections, opts)
+        
+        assert os.path.isfile("Test1.csv"), "Test1.csv does not exists"
+        assert os.path.isfile("Test2.csv"), "Test2.csv does not exists"
+    
+    @testattr(stddist = True)
+    def test_basic_simulation_none_result(self):
+        models, connections = self._load_basic_simulation()
+        
+        opts = {"result_handling":None}
+        
+        master = Master(models, connections)
+       
+        opts["step_size"] = 0.0005
+        res = master.simulate(options=opts)
+        
+        assert res[models[0]]._result_data == None, "Result is not none"
```

Comment:
Possibly other ways to test this? Not too keen on checking internal variables in unit tests.

Reply:
No other good way without doing some restructuring which I'd like to avoid at this point.